src/Webfactory/Slimdump/Doctrine/DummyType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Webfactory\Slimdump\Doctrine;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use LogicException;
+
+class DummyType extends Type
+{
+    public const NAME = 'dummy_type';
+
+    public function convertToPHPValue($value, AbstractPlatform $platform)
+    {
+        throw new LogicException('this should not be called in the first place');
```

Comment:
Maybe we could have a better description in this file on why it exists. Maybe these error messages would be a good place for such a description.

It could be something like `This Doctrine type assumes that the types won't be used  to convert any data – it's just there to allow the rest of DBAL's functionality to work without throwing errors`. What do you think?

Reply:
#109